### PR TITLE
recipes-containers: docker-apps: move to 0.6.0

### PR DESCRIPTION
**Pull request by mike-scott**
- rollback to 0.6.0 to avoid pulling down a cnab container which is used to
  render the docker-compose.yml
- the cnab and invoc containers gets pruned by aktualizr-lite at the end of
  every update meaning they will be re-downloaded on the next update
- this complexity isn't really needed as docker-app project seems to have
  stalled out and may never realize the correct abstractions to make all of
  this worth it.
- simplify, simplify, simplify

Signed-off-by: Michael Scott <mike@foundries.io>

### CODE REVIEW

**Comment by mike-scott** [comment]
> I'm not sure if the unused parameters patch will apply cleanly, but it would be nice to keep that functionality around.

The `matchAndMergeParametersDefinition` function isn't implemented in docker-app for 0.6.0.  Actually, the `internal/commands/parameters.go` file doesn't exist either.  Was there somewhere else I can apply it?

**Comment by mike-scott** [comment]
Fixed an issue with the .bb file where I was still referencing the 2 patches.

**Comment by doanac** [comment]
> The matchAndMergeParametersDefinition function isn't implemented in docker-app for 0.6.0. Actually, the internal/commands/parameters.go file doesn't exist either. Was there somewhere else I can apply it?

I just tested v0.6.0 - it doesn't fail on unused parameters. So we are good to go. This was "fixed" in later versions of docker app I guess

File: meta-lmp-base/recipes-containers/docker-app/docker-app_git.bb [hunk 1]
```diff
@@ -5,26 +5,25 @@ LICENSE = "Apache-2.0"
...
+PV = "v0.6.0"
 
 inherit go goarch
 
 do_compile() {
 	cd ${S}/src/${GO_IMPORT}
 	BUILD_TAG=`git describe --always --abbrev=10`
 	BUILD_COMMIT=`git rev-parse --short HEAD`
-	DOCKER_APP_LDFLAGS="-X ${GO_IMPORT}/internal.GitCommit=${BUILD_COMMIT} \
+	DOCKER_APP_LDFLAGS="-s -w -X ${GO_IMPORT}/internal.GitCommit=${BUILD_COMMIT} \
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-containers/docker-app/docker-app_git.bb]: Why the change here to strip during build? OE already calls strip, which is why the insane skip wasn't required before.

Comment (mike-scott) [reply meta-lmp-base/recipes-containers/docker-app/docker-app_git.bb]: I was getting an stripping error when I built this as a test

Comment (mike-scott) [reply meta-lmp-base/recipes-containers/docker-app/docker-app_git.bb]: @ricardosalveti testing the new version of this PR locally
